### PR TITLE
[DOCS-7670] Update Install with Docker Compose for ACS 23.2

### PR DESCRIPTION
**Pull request by mbujak-hyland**

### CODE REVIEW

**Review by aritrihore — COMMENTED** [review]
LGTM

File: content-services/latest/install/containers/docker-compose.md [hunk 1]
```diff
@@ -266,41 +257,24 @@ Use this information to verify that the system started correctly, and to clean u
...
+        Container acs_trial-activemq-1                                     Removed
+        Container acs_trial-share-1                                        Removed
+        Container acs_trial-digital-workspace-1                            Removed
+        Container acs_trial-alfresco-1                                     Removed
+        Container acs_trial-control-center-1                               Removed
+        Image quay.io/alfresco/service-sync:4.0.1                          Removed
+        ...                               
+        Image quay.io/alfresco/alfresco-content-repository:23.2.1          Stopping                               
+        ...
+        Network acstrial_default                                           Stopping
```
Comment (anxumalo) [inline content-services/latest/install/containers/docker-compose.md:277]: Looks great @mbujak-hyland! I think these need to be written consistently (unless the console output has changed):

- The folder name for testing is defined as `acs_trial`.
- However, `acstrial_*` differs in various places, such as `acstrial_*` (line 277), `acs_trial-*` (lines 260-272), and `acs-trial-*` (other examples).

Comment (mbujak-hyland) [reply content-services/latest/install/containers/docker-compose.md]: Thank you for that, @anxumalo! I fixed it to include only the `acs_trial` folder name.

File: content-services/latest/install/containers/docker-compose.md [hunk 2]
```diff
@@ -161,14 +163,14 @@ Use this information to verify that the system started correctly, and to clean u
 
         ```bash
         docker-compose logs share
-        docker container logs acs-trial_share_1
+        docker container logs acs_trial_share_1
```
Comment (anxumalo) [inline content-services/latest/install/containers/docker-compose.md:166]: This needs to match line 140: `acs_trial-share-1`

**Pull request merged**